src/test/java/CharCounterApplicationTest.java

```diff
@@ -0,0 +1,33 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.mikhail.tarasevich.charcounter.CharCounterApplication;
+import com.mikhail.tarasevich.charcounter.provider.CharCounter;
+import com.mikhail.tarasevich.charcounter.provider.CharCounterImpl;
+import com.mikhail.tarasevich.charcounter.provider.HashMapToStringConverter;
+import com.mikhail.tarasevich.charcounter.provider.HashMapToStringConverterImpl;
+import com.mikhail.tarasevich.charcounter.validator.Validator;
+import com.mikhail.tarasevich.charcounter.validator.ValidatorImpl;
+
+class CharCounterApplicationTest {
+	
+	Validator validator = new ValidatorImpl();
+	CharCounter charCounter = new CharCounterImpl();
+	HashMapToStringConverter hashMapToStringConverter = new HashMapToStringConverterImpl();
+	CharCounterApplication �harCounterApplication = new CharCounterApplication(validator, charCounter, hashMapToStringConverter);
+	
```

Comment:
You must use mock of object at this test, 
please use Mockito here
https://www.baeldung.com/mockito-annotations - about mock injects
https://www.baeldung.com/mockito-junit-5-extension - using with junit 5 
https://www.baeldung.com/mockito-series - all tutarials
https://habr.com/ru/post/444982/ - in Russian